cairo-verifier/src/main.rs

```diff
@@ -0,0 +1,42 @@
+use std::fs;
+
+use clap::Parser;
+use crate::utils::error::VerifierError;
+
+mod commands;
+mod utils;
+
+#[derive(Parser)]
+struct Cli {
+    proof_path: String,
+}
+
+fn main() -> Result<(), VerifierError> {
+    let args: commands::ProverArgs = commands::ProverArgs::parse();
+
+    let res = match args.entity {
+        commands::ProverEntity::Verify(args) => {   
```

Comment:
Verify checks the proof that has been generated, through a file, because it is too heavy to be passed as a parameter right ?